src/main/java/org/sonar/plugins/stash/issue/MarkdownPrinter.java

```diff
@@ -125,57 +138,52 @@ public static String printReportMarkdown(PullRequestRef pr,
       // Issue list
       sb.append("| Issues list |").append(NEW_LINE);
       sb.append("|------------|").append(NEW_LINE);
+      /*
+      List<IssueMetaInformation> uniqueSortedInformation = generalIssues.stream()
+              .map(IssueMetaInformation::from)
+              .distinct()
+              .sorted(Comparator.comparing((item) -> Severity.ALL.indexOf(item.severity()), Comparator.reverseOrder()))
+              .collect(Collectors.toList());

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)